src/Api/Dashboard/DashboardApi.tsx

```diff
@@ -0,0 +1,31 @@
+const serverUrl = "http://18.60.129.53/";
```

Comment:
You shouldn't keep `serverUrl` like this. Ideally these things should come from env variables. Figure out how to create env variables in vite using `.env` files.

Reply:
Created an .env filed and initialized the serverUrl as env variable and imported them when required